.github/workflows/publish-extension.yml

```
@@ -0,0 +1,23 @@
+name: Publish STAC Extension JSON Schema
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/extension')
```

Since we need a tag to run, would it make sense to update `on:` to target tags? Maybe with our given prefix?

Also, should we be more specific?

```suggestion
  push:
    tags:
      - "extension-v*"

jobs:
  deploy:
    runs-on: ubuntu-latest
```

---

👍 looks great, thanks!